build/ci/golangci.yml

```diff
@@ -23,6 +23,13 @@ linters-settings:
     # minimal code complexity to report, 30 by default (but we recommend 10-20)
     min-complexity: 10
 
+  gosec:
+    excludes:
+      - G601 # Implicit memory aliasing of items from a range statement (Go 1.22)
+
+  dupl:
+    threshold: 500
```

Comment:
Increased threshold for duplication detections, to prevent multiple false positives.